Source/WebCore/platform/graphics/GraphicsLayerClient.h

```diff
@@ -108,6 +108,8 @@ class GraphicsLayerClient {
     virtual float pageScaleFactor() const { return 1; }
     virtual float zoomedOutPageScaleFactor() const { return 0; }
 
+    virtual FloatSize viewSize() const { return { }; }
```

Comment:
This confuses me -- what is a 'viewSize' of the GraphicsLayerClient? How does it differentiate from the "layer size" itself, etc.